rules_java_gapic/java_gapic_spring.bzl

```diff
@@ -0,0 +1,135 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_gapic//:gapic.bzl", "proto_custom_library")
```

Comment:
Once we move to spring-cloud-gcp, we would need to add `rules_gapic` as a dependency as well, is it easy to just copy the necessary functionality over?

Reply:
I'm not worrying about how to reference `rules_gapic` in `spring-cloud-gcp`, the question is if we should reference `rules_gapic` or not, which I think it's better not to have it. Because 

1. The code in `rules_gapic` may change, but SpringCodeGen may not really need the change.
2. The end goal is to use something else(Maven or shell script) to generate the code instead of Bazel so the whole process is more streamlined. It may make it harder to migrate if we have additional Bazel dependencies.

However, I do see that moving the code is a little more involved so this is fine at this point. Hopefully we can get rid of Bazel relatively soon.